src/server/routes/contract/extensions/accountFactory/read/predictAccountAddress.ts

```diff
@@ -51,16 +53,16 @@ export const predictAccountAddress = async (fastify: FastifyInstance) => {
       const { chain, contractAddress } = request.params;
       const { adminAddress, extraData } = request.query;
       const chainId = await getChainIdFromChain(chain);
-
-      const contract = await getContract({
+      const factoryContract = await getContractV5({
         chainId,
         contractAddress,
       });
-      const accountAddress =
-        await contract.accountFactory.predictAccountAddress(
-          adminAddress,
-          extraData,
-        );
+
+      const accountAddress = await predictAddress({
+        factoryContract,
+        adminAddress,
+        accountSalt: extraData,
```

Comment:
Should we also "hexify if not hex" here?

Reply:
see above, its done internally